.github/workflows/release-please.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+
+  release-please-app:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: node
+          monorepo-tags: true
+          command: manifest
+          config-file: '.github/release-please-config.app.json'
+          manifest-file: '.github/release-please-manifest.app.json'
+          pull-request-title: 'chore${scope}: release${component} app packages ${version}.'
```

Comment:
Are we missing a space after `release` or is it on purpose?

Reply:
I don't know, tbh.  This was the example from the release-please docs.  We'll try it out and see what happens.  I don't even know how component will be filled in yet!